source/features/pr-branches.tsx

```diff
@@ -67,13 +67,13 @@ function normalizeBranchInfo(data: BranchInfo): {
 	};
 }
 
-function buildQuery(numbers: number[]) {
+function buildQuery(issueIds: string[]) {
 	const {ownerName, repoName} = getOwnerAndRepo();
 
 	return `{
 		repository(owner: "${ownerName}", name: "${repoName}") {
-			${numbers.map(number => `
-				${number}: pullRequest(number: ${String(number).replace('issue_', '')}) {
+			${issueIds.map(id => `
+				${id}: pullRequest(number: ${String(id).replace('issue_', '')}) {
```

Comment:
It’s already a string now, so this can be dropped

Reply:
Duh..Missed it. 😅